src/test/java/org/broadinstitute/hellbender/utils/gcs/BucketUtilsTest.java

```diff
@@ -31,22 +32,36 @@
     }
 
     @Test(groups={"bucket"})
-    public void testIsCloudStorageURL(){
-        Assert.assertTrue(BucketUtils.isCloudStorageUrl("gs://abucket/bucket"));
-        Assert.assertFalse(BucketUtils.isCloudStorageUrl("hdfs://namenode/path/to/file"));
-        Assert.assertFalse(BucketUtils.isCloudStorageUrl("localFile"));
+    public void testIsGcsUrl(){
+        Assert.assertTrue(BucketUtils.isGcsUrl("gs://abucket/bucket"));
+        Assert.assertFalse(BucketUtils.isGcsUrl("hdfs://namenode/path/to/file"));
+        Assert.assertFalse(BucketUtils.isGcsUrl("localFile"));
+        Assert.assertFalse(BucketUtils.isGcsUrl("https://www.somewhere.com"));
+    }
```

Comment:
Can you add a unit test like this for `isHttpUrl()` as well?

Reply:
done